### PR TITLE
Run the code formatter on Mix.Tasks.Compile.Protocols

### PR DESCRIPTION
**Pull request by Nitrino**

### CODE REVIEW

File: lib/mix/lib/mix/tasks/compile.protocols.ex [hunk 1]
```diff
@@ -98,11 +97,11 @@ defmodule Mix.Tasks.Compile.Protocols do
   end
 
   defp consolidation_paths do
-    filter_otp(:code.get_path, :code.lib_dir)
+    filter_otp(:code.get_path(), :code.lib_dir())
   end
 
   defp filter_otp(paths, otp) do
-    Enum.filter(paths, &(not :lists.prefix(&1, otp)))
+    Enum.filter(paths, &not :lists.prefix(&1, otp))
```
Comment (Nitrino) [inline lib/mix/lib/mix/tasks/compile.protocols.ex]: I'm not sure about this line, there is no need braces?

File: lib/mix/lib/mix/tasks/compile.protocols.ex [hunk 2]
```diff
@@ -117,7 +116,7 @@ defmodule Mix.Tasks.Compile.Protocols do
     protocols
     |> Enum.uniq()
     |> Enum.map(&Task.async(fn -> consolidate(&1, paths, output, opts) end))
-    |> Enum.map(&Task.await(&1, 30_000))
+    |> Enum.map(&Task.await(&1, 30000))
```
Comment (Nitrino) [inline lib/mix/lib/mix/tasks/compile.protocols.ex]: Should not we split numbers for readability?

File: lib/mix/lib/mix/tasks/compile.protocols.ex (hunk 1 above)
Comment (whatyouhide) [reply lib/mix/lib/mix/tasks/compile.protocols.ex]: Nope, parens are not needed. I would still prefer `&not(...)` but semantically and syntactically this is correct.

Comment (Nitrino) [reply lib/mix/lib/mix/tasks/compile.protocols.ex]: I also think that `Enum.filter(paths, &not(:lists.prefix(&1, otp)))` would be better, but formatter does not agree :)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**